src/ps_util.c

```diff
@@ -1,6 +1,7 @@
 // Copyright (C) Microsoft Corporation. All rights reserved.
 
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
>stdbool [](http://example.com/codeflow?start=10&length=7)

not used

Reply:
Removed

---
In reply to: [627773732](https://github.com/Azure/c-util/pull/117#discussion_r627773732) [](ancestors = 627773732)